docs/lib/graphqlapi/fragments/flutter/getting-started/30_initapi.md

```diff
@@ -0,0 +1,24 @@
+To initialize the Amplify API category you call `Amplify.addPlugin()` method for each category. To complete initialization call `Amplify.configure()`.
+
+Your code should look like this:
+
+```dart
+import 'package:amplify_core/amplify_core.dart';
+import 'package:amplify_api/amplify_api.dart';
+
+import 'amplifyconfiguration.dart';
+
+class MyAmplifyApp extends StatefulWidget {
+
+    Amplify amplify = Amplify();
+
+    @override
+    void initState() {
+        super.initState();
+
+        AmplifyAPI api = AmplifyAPI();
+        await amplify.addPlugin(apiPlugins: [api]);
```

Comment:
@jamesonwilliams @fjnoyp To Jameson's point: didn't the core refactor result in an API that would be called like this?

```
await amplify.addPlugin(api);
```

Reply:
Its not released yet, so will update all the category docs when that release is going out.